src/Workload/Microsoft.Maui.Sdk/Sdk/AutoImport.props

```diff
@@ -130,7 +130,7 @@
         Condition="Exists('$(ApplicationManifest)')" />
     <AppxManifest
         Include="$(PackageManifest)"
-        Condition="'$(WindowsPackageType)' != 'None' and Exists('$(PackageManifest)')" />
+        Condition="Exists('$(PackageManifest)')" />
```

Comment:
The other half of the fix in #5896 that ran before the .csproj or nugets. This means the only way to set this was via Directory.Build.props or via the CLI - which is not a valid requirement.